README.md

```diff
@@ -4,13 +4,13 @@ Marl is a hybrid thread / fiber task scheduler written in C++ 11.
 
 ## About
 
-Marl is a C++ 11 library that provides a fluent interface for running tasks across a number of threads.
+Marl is a C++ 11 library that provides a fluent interface for running tasks across saveral threads.
 
-Marl uses a combination of fibers and threads to allow efficient execution of tasks that can block, while keeping a fixed number of hardware threads.
+Marl uses a combination of fibers and threads to allow efficient execution of tasks that can block while keeping a fixed number of hardware threads.
```

Comment:
While I'm no english grammar expert, I think the comma here actually helps the reading of the sentence. It breaks up two statements that would otherwise become a bit rambling.